notebooks/oracle2bq/OracleToBigQuery_notebook.ipynb

```diff
@@ -0,0 +1,888 @@
+   "id": "5be3cf87-6d28-4b23-8466-87d3399f7a29",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "\n",
+    "oracle_to_bq_status = []\n",
+    "job_status_url = \"https://dataproc.googleapis.com/v1/projects/{}/locations/{}/batches/{}\"\n",
+    "for job in oracle_to_gcs_jobs:\n",
+    "    auth = \"Bearer \" + token[0]\n",
+    "    url = job_status_url.format(PROJECT,REGION,job)\n",
+    "    headers = {\n",
+    "      'Content-Type': 'application/json; charset=UTF-8',\n",
+    "      'Authorization': auth \n",
+    "    }\n",
+    "    response = requests.get(url, headers=headers)\n",
+    "    oracle_to_bq_status.append(response.json()['state'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1097575d-07c2-4659-a75f-d7e898e3f077",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "statusDF = pd.DataFrame({\"table\" : ORACLETABLE_LIST,\"oracle_to_gcs_job\" : oracle_to_gcs_jobs, \"oracle_to_bq_status\" : oracle_to_bq_status})\n",
+    "statusDF"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0961f164-c7e4-4bb5-80f0-25fd1051147b",
+   "metadata": {},
+   "source": [
+    "## Step 11: Validate row counts of migrated tables from ORACLE to BigQuery"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a3a28fb-3a39-4a10-b92d-0685b351a1b3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "oracle_row_count = []\n",
+    "bq_row_count = []"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25299344-c167-4764-a5d1-56c1b384d104",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# get mysql table counts\n",
+    "DB = sqlalchemy.create_engine(\n",
+    "            sqlalchemy.engine.url.URL.create(\n",
+    "                drivername=ORACLE_DRIVER,\n",
+    "                username=ORACLE_USERNAME,\n",
+    "                password=ORACLE_PASSWORD,\n",
+    "                database=ORACLE_DATABASE,\n",
+    "                host=ORACLE_HOST,\n",
+    "                port=ORACLE_PORT\n",
+    "              )\n",
+    "            )\n",
+    "with DB.connect() as conn:\n",
+    "    for table in ORACLETABLE_LIST:\n",
+    "        results = DB.execute(\"select count(*) from {}\".format(table)).fetchall()\n",
+    "        for row in results:\n",
+    "            oracle_row_count.append(row[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab0e539d-5180-4f5b-915e-35f7ea45e0d3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from google.cloud import bigquery\n",
+    "\n",
+    "# Construct a BigQuery client object.\n",
+    "client = bigquery.Client()\n",
+    "\n",
+    "for table in ORACLETABLE_LIST:\n",
+    "    results = client.query(\"select count(*) from {}.{}\".format(BIGQUERY_DATASET,table))\n",
```

Comment:
`count(*)` in BQ is expensive operation.
Can we get the row counts from information schema instead ?